charts/edge-site/Chart.yaml

```diff
@@ -13,6 +13,6 @@ type: application
 # 1.0.0-alpha.0
 # 1.0.0-alpha.1
 # 1.0.0
-version: "0.0.10"
+version: "0.0.11"
```

Comment:
Look at the comment above this line to see how to make pre-releases. If you call this version `0.0.11-alpha.0` it will get released but not automatically available for upgrade. Helm users have to use the `--devel` flag to install pre-release versions.